addons/website/models/website.py

```diff
@@ -849,12 +849,17 @@ def _handle_views_and_pages(views):
             if dependency_records:
                 model_name = self.env['ir.model']._display_name_for([model])[0]['display_name']
                 dependencies.setdefault(model_name, [])
-                dependencies[model_name] += [{
+                for rec in dependency_records:
```

Comment:
Code looks good :+1: 
I just see in task description that you also have to add a test tour for that
![image](https://user-images.githubusercontent.com/78849981/205276536-960d3af4-40bf-4830-850a-5453bdfd2bef.png)